lib/internal/v8_prof_polyfill.js

```diff
@@ -100,3 +101,30 @@ function versionCheck() {
     }
   }
 }
+
+function macCppfiltNm(out) {
+  // Re-grouped copy-paste from `tickprocessor.js`
+  const FUNC_RE = /^([0-9a-fA-F]{8,16} [iItT] )(.*)$/gm;
```

Comment:
Not a comment, just a note: this matches any hex digit string >= 8 && <= 16 characters long, not just strings of size 8 or 16.  Ditto below.